docker/Dockerfile

```diff
@@ -90,10 +91,15 @@ COPY postgres_config/postgresql.conf .
 # RUN echo "listen_addresses = '*'" >> postgresql.conf
 COPY postgres_config/pg_hba.conf .
 
-# --- INITIALISE DATABASE (core templates)
+# --- INITIALISE DATABASE (default: core templates)
+ARG INITIAL_SNAPSHOT=""
+WORKDIR /usr/src/conforma-server/database/_snapshots
+COPY snapshots/${INITIAL_SNAPSHOT}/ ./${INITIAL_SNAPSHOT}
```

Comment:
We copy the specified snapshot into the container's "snapshots" folder, so it can be loaded with `useSnapshot`

(Note: for now, snapshot (unzipped) needs to be places in `/docker/snapshots`)